### PR TITLE
BUG: Make more strings translatable

### PR DESCRIPTION
**Pull request by lassoan**
Marked strings that were found to be non-translatable in https://github.com/Slicer/SlicerLanguagePacks/issues/41

### CODE REVIEW

File: Modules/Scripted/SampleData/SampleData.py [hunk 1]
```diff
@@ -856,51 +861,53 @@ def downloadFile(self, uri, destFolderPath, name, checksum=None):
...
 
             if algo is not None:
-                self.logMessage('<b>Verifying checksum</b>')
+                self.logMessage(_('Verifying checksum'))
                 current_digest = computeChecksum(algo, filePath)
                 if current_digest != digest:
-                    self.logMessage(f'<b>Checksum verification failed. Computed checksum {current_digest} different from expected checksum {digest}</b>')
+                    self.logMessage(
+                        _('Checksum verification failed. Computed checksum {currentChecksum} different from expected checksum {expectedChecksum}').format(
+                        curentChecksum=current_digest, expectedChecksum=digest))
```
Comment (jcfr) [inline Modules/Scripted/SampleData/SampleData.py:878]:
```suggestion
                            curentChecksum=current_digest, expectedChecksum=digest))
```

File: Modules/Scripted/SampleData/SampleData.py [hunk 2]
```diff
@@ -856,51 +861,53 @@ def downloadFile(self, uri, destFolderPath, name, checksum=None):
         (algo, digest) = extractAlgoAndDigest(checksum)
         if not os.path.exists(filePath) or os.stat(filePath).st_size == 0:
             import urllib.request, urllib.parse, urllib.error
-            self.logMessage(f'<b>Requesting download</b> <i>{name}</i> from {uri} ...')
+            self.logMessage(_('Requesting download {name} from {uri} ...').format(name=name, uri=uri))
```
Comment (jcfr) [inline Modules/Scripted/SampleData/SampleData.py:864]: Is the removal of `<b>` intentional ?

Comment (lassoan) [reply Modules/Scripted/SampleData/SampleData.py:864]: Yes, it is part of `logMessage` method now (by default it is added around the text).

File: Modules/Scripted/SampleData/SampleData.py [hunk 3]
```diff
@@ -624,9 +626,9 @@ def downloadFileIntoCache(self, uri, name, checksum=None):
             try:
                 os.makedirs(destFolderPath, exist_ok=True)
             except:
-                self.logMessage('<b>Failed to create cache folder %s</b>' % destFolderPath, logging.ERROR)
+                self.logMessage(_('Failed to create cache folder {path}').format(path=destFolderPath), logging.ERROR)
```
Comment (jcfr) [inline Modules/Scripted/SampleData/SampleData.py:629]: Should the parameter `bold=True` be passed where relevant then ?

**Review by jcfr — DISMISSED** [review]

**Comment by lassoan** [comment]
Thanks for the review @jcfr.

@mhdiop I'll merge it now to allow adding the new translatable strings to weblate, but if you find anything that I've missed then please submit a pull request.

**Pull request merged**